### DISCUSSION TITLE
Inquiry on Optimizing Cerebellum Stimulation

### DISCUSSION THREAD

**Discussion by ryokisasaki** [simnibs/simnibs#421] [9. Results and analysis] [answered]
Dear SimNIBS experts,

I am currently working on optimizing a tDCS electrode montage and intensity to target the cerebellum. I have specified a location on the cerebellum, and after running the optimization code, the cerebellum did not appear in the resulting Gmsh file. However, when I manually ran a basic simulation through the SimNIBS interface, the cerebellum was visible. Is there a way to optimize simulation specifically for the cerebellum? I reviewed the tutorial but could not find information related to this issue. I would greatly appreciate any guidance or solutions you could provide. 

Best regards,
Ryoki

![Cerebellum_opt](https://github.com/user-attachments/assets/93abca84-ae15-4f79-b9e8-21aa1af332fe)

**Comment by diedrim** [comment, marked as answer]
Please have a look here: https://simnibs.github.io/simnibs/build/html/documentation/sim_struct/tdcsleadfield.html
In your case, you'd have to set
```
tdcs_lf.interpolation = None
tdcs_lf.tissues = [2]
```

**Reply by ryokisasaki** [reply]
Dear Diedrim,

Thank you so much for your guidance. It was so helpful.

Best regards,
Ryoki
